modules/oe_translation_remote/tests/src/Functional/RemoteTranslationTest.php

```diff
@@ -36,4 +37,66 @@ public function testRemoteTranslationRoute(): void {
     $this->assertSession()->pageTextContains('The remote translations overview page.');
   }
 
+  /**
+   * Test the remote translation provider configuration.
+   */
+  public function testRemoteTranslationProvider(): void {
```

Comment:
Name the tests a bit better, this is very vague and the test doesn't do this. The test tests that creation of a translation provider config. So something like `testRemoteTranslationProviderForm` is better.